tajo-algebra/src/main/java/org/apache/tajo/algebra/CreateTable.java

```diff
@@ -419,9 +420,7 @@ public Object clone() throws CloneNotSupportedException {
       hash.quantity = quantity;
       if (specifiers != null) {
         hash.specifiers = new ArrayList<>();
-        for (PartitionSpecifier specifier : specifiers) {
-          hash.specifiers.add(specifier);
-        }
+        hash.specifiers.addAll(specifiers.stream().collect(Collectors.toList()));
```

Comment:
Actually in a case like this, I think it's sufficient and more clear.

```java
hash.specifiers.addAll(specifiers);
```

or more simply,

```java
hash.specifiers = new ArrayList<>(specifiers);
```

Anyway, I think stream doesn't matter.
Because it is simple shallow copy, there are many ways.